DBM-Naxx/ConstructQuarter/Gluth.lua

```diff
@@ -3,26 +3,39 @@ local L		= mod:GetLocalizedStrings()
 
 mod:SetRevision(("$Revision: 2869 $"):sub(12, -3))
 mod:SetCreatureID(15932)
+mod:SetUsedIcons(7, 8)
 
 mod:RegisterCombat("combat")
 
 mod:EnableModel()
 
 mod:RegisterEvents(
-	"SPELL_DAMAGE"
+	"SPELL_DAMAGE",
+	"SPELL_AURA_APPLIED",
+	"SPELL_AURA_REMOVED"
 )
 
 
 local warnDecimateSoon	= mod:NewSoonAnnounce(54426, 2)
 local warnDecimateNow	= mod:NewSpellAnnounce(54426, 3)
+local specwarnFleshRip	= mod:NewSpecialWarning("SpecialWarningFleshRip")
+local warnFleshRip		= mod:NewTargetAnnounce(40199, 2)
 
 local enrageTimer		= mod:NewBerserkTimer(420)
 local timerDecimate		= mod:NewCDTimer(104, 54426)
 
+mod:AddBoolOption("SetIconOnFleshRipTarget", true)
+
+local fleshripIcons = {}
+
 function mod:OnCombatStart(delay)
-	enrageTimer:Start(420 - delay)
 	timerDecimate:Start(110 - delay)
 	warnDecimateSoon:Schedule(100 - delay)
+	if mod:IsDifficulty("heroic25") then
+		enrageTimer:Start(480 - delay)
+	else
+	enrageTimer:Start(420 - delay)
```

Comment:
Uhh missed indent

Reply:
why would you comment on stuff that is already in the core?! all i did is copy paste to fix the differences that didnt let us squash and merge